### ISSUE TITLE
'Any_*' matchers for expect

### ISSUE THREAD

**Issue by twig2let** [rokucommunity/rooibos#70]
It's not always necessary to expect on the value of each parameter. For example it's cumbersome to expect the value of a string argument containing dynamic text.

Any matchers would provide additional flexibility and simplicity when expecting in a test. 

```
m.ExpectOnce(m.test.SUT.method, "methodName", [m.AnyString, true])
```

For example: https://static.javadoc.io/org.mockito/mockito-core/1.9.5/org/mockito/Matchers.html

**Comment by twig2let** [comment]
Happy to pick this up, i'm just super low on time atm. Aren't we all!

**Comment by georgejecook** [comment]
we already have this.

`m.ignoreValue`

https://github.com/georgejecook/rooibos/blob/master/docs/index.md#skipping-value-assertion-with-mignorevalue

do you specifically want matchers for anyString, anyBool, etc, coz if so, then yes - please run with it; but if it's not needed we can close the issue.

btw, for the longest time I've wanted a custom matcher function. Personally, I'll go ahead and add it today.

**Comment by georgejecook** [comment]
implemented, as of 0.3.2
